### PR TITLE
Allow overflowed SVGs scroll-able and restructure `index.html`

### PR DESCRIPTION
**Pull request by aaronthangnguyen**
# Description

PR redesigns `index.html` by allowing `overflow: scroll` in each sample and restructuring page layout.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Visual testing via `example/index.htm`
- [x] Jest `npm run test`

**Prompt when `qviz.min.js` is unavailable**
Screen Shot 2022-05-19 at 11 53 56 AM

**New layout**
Screen Shot 2022-05-19 at 11 52 49 AM

**Overflowed SVGs are now horizontally scroll-able**

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by anpaz — APPROVED** [review]
To be clear, these changes only show how the user should setup their container if they want overflow scroolbars show. qviz will not automatically add scrolls to itself (which is the correct behavior).

File: example/index.html [hunk 1]
```diff
@@ -2,22 +2,50 @@
...
+            <div id="sample" class="container"></div>
+            <h4>3. Teleport</h4>
+            <div id="teleport" class="container"></div>
+            <h4>4. Grover</h4>
+            <div id="grover" class="container"></div>
+        </div> 
+    </main>
 </body>
-</html>
+</html>
```
Comment (anpaz) [inline example/index.html:51]:
```suggestion
</html>

```
nox is complaining about not having an eol in the last line of the file.

Comment (anpaz) [reply example/index.html:51]: @aaronthangnguyen could you fix this?

Comment (aaronthangnguyen) [reply example/index.html]: Of course. I accepted the commit suggestion. Did it fix the issue?

**Pull request merged**